pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -296,6 +305,18 @@ func (m *manager) GetPodTopologyHints(pod *v1.Pod) map[string][]topologymanager.
 	return m.policy.GetPodTopologyHints(m.state, pod)
 }
 
+func (m *manager) GetAllocatableCPUs() []int64 {
+	// please note that 'assignableCPUs' is the old internal naming which predated
+	// the introduction of the `GetAllocatableCPUs` API.
+	// We keep the old name internally for consistency within the cpumanager code.
```

Comment:
I don't think this is necessary -- let's just rename it. Also, with my suggestion above all of the code below this goes away, and we just return `m.allocatableCPUs`.

Reply:
agreed.